bigquery_etl/glam/templates/logical_app_id/org_mozilla_fenix_glam_beta__view_clients_daily_histogram_aggregates_v1.sql

```diff
@@ -7,7 +7,7 @@ WITH extracted AS (
   FROM
     `moz-fx-data-shared-prod`.glam_etl.org_mozilla_fenix__view_clients_daily_histogram_aggregates_v1
   WHERE
-    `moz-fx-data-shared-prod`.udf.fenix_build_to_datetime(app_build_id) < date "2020-07-03"
+    mozfun.norm.fenix_app_info('org_mozilla_fenix', app_build_id).channel = 'beta'
```

Comment:
This is an autogenerated file, it actually needs to be updated in https://github.com/mozilla/bigquery-etl/blob/f25a096536a10effd5f0f7b09051da9484a76234/bigquery_etl/glam/generate.py#L122-L127. The script is expected to have a callback of a single argument though.

Reply:
^ This is inaccurate, since the mapping only affects the final published form of the dataset into csv. I remember creating these views by hand.